base/src/main/scala/co/uproot/abandon/Process.scala

```diff
@@ -29,14 +29,14 @@ case class DetailedTransaction(name: AccountName, delta: BigDecimal, commentOpt:
 class AccountState {
 
   private var _amounts = Map[AccountName, BigDecimal]() // initAmounts
-  private var _txns = Seq[DetailedTransaction]() // initTxns
-  private var _txnGroups = Seq[TxnGroup]() // initTxns
+  private var _txns = Seq[DetailedPost]() // initTxns
+  private var _txnGroups = Seq[PostingGroup]() // initTxns
```

Comment:
The names of vars can be `_posts` and `_postGroups`. Also the comments (initTxns) can be removed.